functions/src/courses/Course.model.ts

```diff
@@ -28,4 +28,8 @@ export interface CourseDetails
    * @example "ELEC260"
    */
   formally?: string;
+  /**
+   * Course hours
+   */
+  hoursCatalogText?: { lecture: string; lab: string; tutorial: string };
```

Comment:
This isn't needed anymore.

Reply:
@aomi when I delete this line, /funcitons/build/routes.ts file and functions/build/swagger.json file automatically changed. is it okay?

Reply:
Yeah that's expected and what we want :+1: